kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/metrics/MetricsCollector.java

```diff
@@ -330,6 +325,11 @@ private String collectMetrics(String metricsPodIp, String podName) throws Interr
 
         LOGGER.info("Metrics collection for Pod: {}/{}({}) from Pod: {}/{} finished with return code: {}", namespaceName, podName, metricsPodIp, namespaceName,
                 scraperPodName, ret);
+
+        if (ret != 0) {
+            throw new IOException("Cannot collect the metrics using " + executableCommand + " command");
```

Comment:
it is not a IOException, so don't suggest it is 

```suggestion
            throw new RuntimeException("Cannot collect the metrics using " + executableCommand + " command");
```